### DISCUSSION TITLE
Quasar table with Floating action button

### DISCUSSION THREAD

**Discussion by akashramtohul4** [quasarframework/quasar#9710] [CLI - PWA mode]
Hello Friends 

I want to implement quasar table with Floating Action Button. am not able to fine the proper way of doing it 
Can someone please help . I have shared the link of floating action button below
https://quasar.dev/vue-components/floating-action-button#introduction 

I have been ale to add the Button but its not working on individual row 
![image](https://user-images.githubusercontent.com/84211129/122560930-41840980-d052-11eb-8e4f-4f353e8b0ac3.png)

Thanks

**Comment by ontwikkelfabriek** [comment]
@akashramtohul4 

To receive better support , please create a codepen that shows you problem. The image does not tell what exactly is not working.
